doc/changelog.rst

```diff
@@ -11,6 +11,71 @@ Jump to :ref:`SmartRedis Changelog <changelog>`
 SmartSim
 ========
 
+0.4.1
+-----
+
+Released on June 24, 2022
+
+Description:
+This release of SmartSim introduces a new experimental feature to help make
+SmartSim workflows more portable: the ability to run simulations models in a
+container via Singularity. This feature has been tested on a small number of
+platforms and we encourage users to provide feedback on its use.
+
+We have also made incremental improvements in a variety of areas: being able to
+load scripts and machine learning models into the database directly from
+SmartSim driver scripts, the ability to use `KeyDB` instead of `redis-server`,
+and support for using OpenMPI to launch workflows. Reducing the barrier to entry
+by providing more extensive tutorials and improving the documentation was
+another key focus of this release cycle. Interested newcomers and users to
+SmartSim can now download a docker image to run many of the tutorials described
+in the documentation.
+
+
+Launcher improvements
+
+    - New methods for RunSettings (_SmartSim-PR166) (_SmartSim-PR170)
```

Comment:
Should we say "OpenMPI RunSettings"

Reply:
I think it's more accurate to say "including OpenMPI"?

Reply:
Oh wait, I think these PRs are really just the additional setter methods for generic RunSettings. However, I should include the OpenMPI-related PRs.